wfl/Profiles/RootProfile.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Hypermedia.Affordances;
+using Hypermedia.Transforms;
+using Hypermedia;
+using Symphono.Wfl.Controllers;
+using Symphono.Wfl.Models;
+
+namespace Symphono.Wfl.Profiles
```

Comment:
Personal opinion here, I'd suggest creating a child namespace of `Hypermedia`. In other projects we've also had mapping profiles, validation profiles, etc.  That too gets complicated, so any thoughts on putting all the files for a "type" in the same place -- for example, having Root, RootProfile, RootController, etc all in one spot (this is a discussion question by the way)?